### ISSUE TITLE
Different sizes 

### ISSUE THREAD

**Issue by delivey** [AmreshSinha/Spotify-Cards-API#12]
**DISCUSSION NEEDED**

So the official Spotify promo cards website has a feature where you can select the size.
![image](https://user-images.githubusercontent.com/35540227/135894426-5a2eb012-4837-4072-a194-9b93963d5468.png)
I think we could do something like this.

We would need to add a total of 2 styles:
- Square
- Portrait

What do you think?

(Don't assign me just yet, if after the discussion we agree on adding this, I can experiment with something like this)

**Comment by AmreshSinha** [comment]
Yeah I had plans for this. I was experimenting other stuffs first like podcasts, playlist, etc. You can try experimenting and creating a layout by taking up the coordinates of image and text from different aspect ratio cards from their site. I am currently working on playlist and podcast part. (Pss. You can use Paint for finding coordinates)

**Comment by AmreshSinha** [comment]
Make a diff-aspect-ratio branch and there you can make changes :+1:

**Comment by delivey** [comment]
My square implementation: 
![image](https://user-images.githubusercontent.com/35540227/136669002-c55665b5-8186-45c8-9de3-666635a8d272.png)


Spotify square implementation: 
![image](https://storage.googleapis.com/promocards-byspotify-com.appspot.com/sharing/49e1da3e31c2493c252428e07d3aba87c0c3d03f.png)

Let me know if this looks good, and if it does I will push.

**Comment by AmreshSinha** [comment]
@delivey
